### ISSUE TITLE
Ember 2.7 support

### ISSUE THREAD

**Issue by chinthakagodawita** [mitchlloyd/ember-islands#23]
When trying to run this with a brand new Ember (CLI) 2.7 project, I get the following error:

```
ember-islands.js:51 Uncaught TypeError: componentLookup.lookupFactory is not a function
```

Downgrading to 2.6 (via `bower.json` but still keeping `ember-cli` at 2.7) seems to fix it.

Full error:

![error](https://cloud.githubusercontent.com/assets/1452384/17165678/6f47a5ac-5415-11e6-82ce-4ac2dbe6824e.png)

**Comment by mitchlloyd** [comment]
Thanks for opening. I created a PR to support the new 2.7 release here: https://github.com/mitchlloyd/ember-islands/pull/24.

**Comment by mitchlloyd** [comment]
At fixed version has been published as 1.0.1.

**Comment by chinthakagodawita** [comment]
Brilliant, thanks!
